docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,114 @@
+=============
+Accessibility
+=============
+
+All aspects of Django must be compatible with our accessibility standards,
+and with the specific support targets we aim for.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+Django aims for conformance with `WCAG 2.2`_ at the AA level.
+WCAG is the most well established standard for web accessibility. AA-level
+guidelines are the most common legal compliance target around the world.
+
+Wherever applicable, we also attempt to follow other best practices, such as:
+
+* WCAG 2.2 AAA-level guidelines.
+* The upcoming `WCAG 3.0`_ guidelines.
+* `ATAG 2.0`_ guidelines for authoring tools.
+
+.. note::
+    Do you want to learn more about accessibility without diving straight into
+    standards? We recommend `The A11Y Project`_,
+    a community-driven effort to make digital accessibility easier.
+
+.. _WCAG 2.2: https://www.w3.org/TR/WCAG22/
+.. _WCAG 3.0: https://www.w3.org/TR/wcag-3.0/
+.. _ATAG 2.0: https://www.w3.org/TR/ATAG20/
+.. _The A11Y Project: https://www.a11yproject.com/
+
+.. _accessibility-support-targets:
+
+Accessibility support targets
+=============================
+
+Beyond standards, we want to make sure Django actually works for people using
+a wide range of input devices and assistive technologies.
+The best way to do this effectively is to take accessibility considerations
+into account as part of designing features. If in doubt, consult
+with users who rely on assistive technologies or with accessibility experts.
+
+.. _accessibility-testing-baseline:
+
+Accessibility testing baseline
+------------------------------
+
+ Always test user interface changes with:
+
+* Keyboard-only navigation
+* Automated checks in the `Accessibility Insights`_ browser extension,
+  or an equivalent tool with the `Axe`_ checker.
+
+Where appropriate, also test with:
+
+* Touch-only navigation
+* 400% browser zoom
+* forced-colors mode DevTools emulation
+
+.. note::
+    How much should you test? Devise the UI with accessibility in mind,
+    and the testing will only be needed as a final check. For more complex
+    interfaces, confer with other contributors to decide on testing targets.
+
+.. _assistive-technologies:
+
+Assistive technologies
+----------------------
+
+Where more extensive testing is appropriate,
+here are free assistive technologies we recommend testing with:
+
+* `NVDA`_ on Windows with Firefox ESR
+* `VoiceOver`_ on macOS with Safari
+* `Narrator`_ on Windows with Microsoft Edge
+* Windows Magnifier and macOS Zoom
+* Windows Speech Recognition and macOS Voice Control
+* VoiceOver on iOS, or TalkBack on Android
+* `Contrast themes`_ on Windows
+
+Where expert help is available, also consider testing with:
+
+* `JAWS`_ on Windows
+* `Dragon`_ on Windows
```

Comment:
These feel essential to me because they are very popular, but I chose to keep them separate because:

- Their licenses cost a lot of money
- They’re Windows-only